airbyte-integrations/connectors/source-github-singer/source_github_singer/source.py

```diff
@@ -104,6 +97,19 @@ def get_sync_mode_overrides(self) -> Dict[str, SyncModeInfo]:
             overrides[stream_name] = SyncModeInfo(supported_sync_modes=[SyncMode.full_refresh])
         return overrides
 
+    def get_excluded_streams(self) -> List:
```

Comment:
It's not obvious in the code why we're doing this so let's leave the copntext

```suggestion
    def get_excluded_streams(self) -> List:
    # The below streams are only synced if their parent stream is synced. For example, review_comments is not synced unless the pull_requests stream is selected. If a user tries to sync the child without the parent, however, the tap process succeeds without any exceptions, but no output is emitted for either stream, which is a pretty opaque user experience. So for now we will exclude the child stream until Airbyte has a way of enforcing dependencies between streams.
```